docker/create-buckets-aws/scripts/bucket-setup.py

```diff
@@ -0,0 +1,96 @@
+
+import boto3
+import os
+
+class S3Client:
+    def __init__(self, host, port, access_key, secret_key, data_bucket, state_bucket, sentinel_key):
```

Comment:
see if you can run pylint or some other linter/checker so you can get a PEP8 check on your python code.